<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurnGrayDialogBase</class>
 <widget class="QDialog" name="TurnGrayDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turn Gray</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QgsColorButtonV2" name="mColorButtonForeground">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="color">
    <color>
     <red>127</red>
     <green>127</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="defaultColor">
    <color>
     <red>127</red>
     <green>127</green>
     <blue>127</blue>
    </color>
   </property>
  </widget>
  <widget class="QgsColorButtonV2" name="mColorButtonBackground">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="color">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_composer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>TurnGrayDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>TurnGrayDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
